frontend/interfaces/software.ts

```diff
@@ -198,6 +198,7 @@ export const SOURCE_TYPE_CONVERSION = {
   portage_packages: "Package (Portage)",
   rpm_packages: "Package (RPM)",
   yum_sources: "Package (YUM)",
+  fleetd_pacman_packages: "Package (pacman)",
```

Comment:
We should just use `pacman_packages` here. The fact that we are using a `fleetd` table for now is internal.